src/bot.js

```diff
@@ -128,17 +123,17 @@ class HangoutsChatBot extends Adapter {
    *     </ul>
    */
   reply(envelope, ...strings) {
+    console.log(envelope, strings)
```

Comment:
Please use Hubot's logger (`this.robot.logger.debug`).

Reply:
I'll delete that line. I was just debugging.